ocs-api/scripts/agent-install-wrapper.sh

```diff
@@ -60,18 +60,38 @@ if [ "${pkgsFrom%%:*}" = 'css' ]; then
     echo "Downloading $agentInstallRemotePath ..."
     httpCode=`curl -sSL -w "%{http_code}" -u "$deviceOrgId/$nodeAuth" --cacert agent-install.crt -o agent-install.sh "$agentInstallRemotePath"`
     if [ $? -ne 0 -o "$httpCode" != '200' ]; then
-        echo "~~~~~~~~~~~~~~~~\nError downloading $agentInstallRemotePath: httpCode=$httpCode\n~~~~~~~~~~~~~~~~"
+        echo "~~~~~~~~~~~~~~~~"
+        echo "Error downloading $agentInstallRemotePath: httpCode=$httpCode"
+        echo "~~~~~~~~~~~~~~~~"
+        exit 2
+    fi
+elif [ "$pkgsFrom" = 'https://github.com/open-horizon/anax/releases/latest/download' ]; then 
+    # Download the horizon-agent-edge-cluster-files.tar.gz package that contains install-agent.sh script
+    agentFilesRemotePath="$pkgsFrom/horizon-agent-edge-cluster-files.tar.gz"
+    echo "Downloading $agentFilesRemotePath ..."
+    wget "$agentFilesRemotePath"
+    if [ $? -ne 0 ]; then
+        echo "~~~~~~~~~~~~~~~~"
+        echo "Error downloading $agentFilesRemotePath"
+        echo "~~~~~~~~~~~~~~~~"
         exit 2
     fi
-else   # $pkgsFrom==https://github.com/open-horizon/anax/releases/* but $cfgFrom is likely css:
-    # It is a URL like https://github.com/open-horizon/anax/releases/latest/download, just add agent-install.sh to the end
+    # Extract the package
+    tar -zxvf horizon-agent-edge-cluster-files.tar.gz
+elif echo "$pkgsFrom" | grep -q '^https://github.com/open-horizon/anax/releases/'; then
+    # It is a URL like https://github.com/open-horizon/anax/releases/tag/vX.Y.Z/download, just add agent-install.sh to the end
```

Comment:
do we need this condition `echo "$pkgsFrom" | grep -q '^https://github.com/open-horizon/anax/releases/'`? I think it should be just `else` here.